src/repositories/shoppingCart.js

```diff
@@ -1,29 +1,29 @@
 import EventBus from "@/event-bus";
 import { NEW_PRODUCT_IN_SHOPPING_CART_EVENT } from "@/constants";
 
+// TODO: move this in main (to prevent any constraint on how it is used) ?
 export const shoppingCartPlugin = {
   install: function(Vue) {
-    Vue.prototype.$shoppingCartState = shoppingCartState;
-    Vue.prototype.$shoppingCartMutators = shoppingCartMutators;
+    Vue.prototype.$shoppingCart = shoppingCart;
   }
 };
 
-export const shoppingCartState = {
-  state: {
+export const state = {
     products: []
-  }
 };
 
-export const shoppingCartMutators = {
-  add: product => {
+export const shoppingCart = {
+  state: state,
+  add(product) {
+    // (i) On doit pouvoir gérer ça a la main facilement sans lib externe a mon avis
```

Comment:
c'est pas une lib externe, c'est intégré dans vue.

Reply:
https://medium.com/@andrejsabrickis/https-medium-com-andrejsabrickis-create-simple-eventbus-to-communicate-between-vue-js-components-cdc11cd59860

Reply:
Ah oui, en effet ! Je ne connaissais pas, c'est pas mal !